packages/codegen-ui-react/lib/__tests__/__snapshots__/react-studio-template-renderer-helper.test.ts.snap

```diff
@@ -0,0 +1,104 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`react-studio-template-renderer-helper transpile fails to transpile with ScriptTarget ESNext 1`] = `"Unhandled Codegen Error Occurred - \\"TypeError: Cannot read property 'length' of undefined\\\\n    at iterateCommentRanges (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:10403:45)\\\\n    at reduceEachLeadingCommentRange (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:10494:16)\\\\n    at Object.getLeadingCommentRanges (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:10509:16)\\\\n    at Object.getJSDocCommentRanges (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:15198:16)\\\\n    at addJSDocComment (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:30663:42)\\\\n    at parseSourceFileWorker (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:30636:34)\\\\n    at Object.parseSourceFile (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:30468:26)\\\\n    at Object.createSourceFile (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:30266:29)\\\\n    at Object.getSourceFile (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/@typescript/vfs/src/index.ts:504:16)\\\\n    at findSourceFileWorker (/Users/alharris/workplace/amplify-codegen-ui/packages/codegen-ui-react/node_modules/typescript/lib/typescript.js:113284:29)\\""`;
```

Comment:
Actually, planning to throw this as an InvalidInputException, instead of InternalError.